app/black_spots/views.py

```diff
@@ -74,3 +76,136 @@ def get_queryset(self):
             BlackSpotConfig.objects.create()
             config = BlackSpotConfig.objects.all().order_by('pk').first()
         return BlackSpotConfig.objects.filter(pk__in=[config.pk])
+
+
+class EnforcerAssignmentViewSet(drf_mixins.ListModelMixin, viewsets.GenericViewSet):
+    """
+    ViewSet for enforcer assignments
+
+    Enforcer assignments do not currently have an associated model in the database, and are
+    instead based direcly on black spots, so this is a list-only ViewSet. This was created
+    as its own ViewSet to make it easier if we ever decide to create a db model for
+    enforcer assignments.
+    """
+    queryset = BlackSpot.objects.all()
+    serializer_class = EnforcerAssignmentSerializer
+    filter_class = EnforcerAssignmentFilter
+    permission_classes = (IsAdminOrReadOnly, )
+
+    def get_required_param(self, key, request):
```

Comment:
We've run into this before; it's really awkward to use different serializers for input and output. I _think_ that it should be possible to do something like this: http://stackoverflow.com/a/17552724 , but I seem to recall that I've tried that and it didn't work for me. But this case might be different than whatever case I was working on, so the approach might work here.